sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/spanner/changestreams/mapper/ChangeStreamRecordMapper.java

```diff
@@ -197,13 +209,21 @@ public class ChangeStreamRecordMapper {
    * @return a {@link List} of {@link ChangeStreamRecord} subclasses
    */
   public List<ChangeStreamRecord> toChangeStreamRecords(
-      PartitionMetadata partition, Struct row, ChangeStreamResultSetMetadata resultSetMetadata) {
-    return row.getStructList(0).stream()
+      PartitionMetadata partition,
+      ChangeStreamResultSet row,
```

Comment:
Since we are calling this method with a `ChangeStreamResultSet` now, this is no longer a row, but a `resultSet`. We make it a row by either calling `resultSet.getPgJsonb(0)` or `resultSet.getCurrentRowAsStruct()`. Could you change the variable name here?